src/models/dashboard-items.js

```diff
@@ -94,27 +99,32 @@ const dataMapper = (dataList, filterList) =>
 
 export function itemsMetadataConstructor(dashboardItemsMetadata) {
     const { dataElements, indicators, antigenCategory } = dashboardItemsMetadata;
-    const {
-        tablesDataCodes: { vaccinesTable, qsIndicatorsTable },
-        chartsDataCodes: { indicatorChart, utilizationRateChart },
-    } = dashboardItemsConfig;
+    const { tablesDataCodes, chartsDataCodes } = dashboardItemsConfig;
```

Comment:
When I first tested the code against my local dhis2, a couple of charts did not work, some /analytics calls returned a 409. I remember that there was a typo in the metadata (`UTILITZATION`). Once I fixed, the dashboards work.

However, I think we should fail on the saving instead of creating data with problems. My typical approach is to use `getOrFail` from lodash-mixins.js any time I want to be sure I get a value. So instead of `arrayOrObject[key]` (which may return `undefined`) we  use `_(arrayOrObject).getOrFail(key)` whenever we can't accept `undefined` as an asnwer. 

For now, this shows an error on the console. On the future, we will show an error without extra work in the inner code, just using a global exception catcher (https://github.com/EyeSeeTea/d2-ui-components/issues/15)

(This already comes from #70, but no problem to fix it here.)

Reply:
I agree completely. When you have a few min can you take a look at https://github.com/EyeSeeTea/vaccination-app/pull/73/files#diff-acf0ee4d45f0addd04ae3e13f8c87b6cR265 and explain a bit how you would implement getOrFail in these situations?

Reply:
Using the existing getOrFail:

```
const { antigenCategory, ...elements } = _(["antigenCategory", "someKeyIWant", "someOtherKeyIalsoWant"])
  .map(key => [key, _(dashboardItemsElements).getOrFail(key)])
  .fromPairs()
  .value()
```
Creating a new abstraction (safe version of _.pick):

```
const { antigenCategory, ...elements } = _(dashboardItemsElements)
  .pickOrFail(["antigenCategory", "someKeyIWant", "someOtherKeyIalsoWant"])
```

Reply:
great, so you specify which keys wil be in elements so it checks for all of them, that was my doubt. thanks!